datahub-web-react/src/app/entityV2/shared/tabs/Incident/AcrylComponents/IncidentView.tsx

```diff
@@ -200,7 +203,11 @@ export const IncidentView = ({ incident }: { incident: IncidentTableRow }) => {
             </DescriptionSection>
             <DetailsSection>
                 <DetailsLabel>Category</DetailsLabel>
-                <CategoryText>{incident?.type && getCapitalizeWord(incident?.type)}</CategoryText>
+                <CategoryText>
+                    {getCapitalizeWord(
```

Comment:
Move the logic to line 190:

const categoryName = getCapitalizeWord(...

{categoryName}